### PR TITLE
adding first round of 2019 forum sponsors

### LINKED ISSUE #1807: Adding First Sponsors to Forum Page
@KCMack Preview ready here:
https://preview.tnris.org/ellwood/133a048e5e8022952aa2612e03978808827287dd/texas-gis-forum/2019/

### PR DESCRIPTION
**Pull request by christoball**
closes #1807

### CODE REVIEW

File: templates/events/texas-gis-forum/2019/main.html [hunk 1]
```diff
@@ -111,7 +111,7 @@ <h3>Job Board</h3>
 {% endblock contents %}
 
 {% block contents-two %}
-<!-- <section class="forum-sponsors">
+<section class="forum-sponsors">
```
Comment (jwhaney) [inline templates/events/texas-gis-forum/2019/main.html]: indent here

File: templates/events/texas-gis-forum/2019/home.html [hunk 2]
```diff
@@ -58,6 +58,7 @@
         {% include "events/texas-gis-forum/2019/map.html" %}
       </div>
   </section>
-<section class="forum-sponsors">
-</section>
+  <section class="forum-sponsors">
+      {% include "events/texas-gis-forum/2019/sponsors.html" %}
```
Comment (jwhaney) [inline templates/events/texas-gis-forum/2019/home.html]: indent and alignment is off

File: content/markdown/texas-gis-forum/2019/index.md [hunk 3]
```diff
@@ -24,6 +24,11 @@ buttontext: Register Now!
 
 {% include "events/texas-gis-forum/2019/registration.html" %}
 
+
+<h3>Thanks to Our Platinum Sponsors</h3>
+<p>    <a class="welcome-logo-2019" href="http://www.surdex.com/"><img alt="Surdex geospatial logo and link" src="{{ m.link('static/images/vendors/surdex_logo.png')}}"></a>  <a class="welcome-logo-2019" href="http://appgeo.com"><img  alt="appgeo logo and link to website" src="{{ m.link('static/images/texas-gis-forum/2015/sponsors/appgeo_logo.png')}}"></a>
```
Comment (jwhaney) [inline content/markdown/texas-gis-forum/2019/index.md]: this section is pretty jumbled and disorganized. i would split each element out on its own line and indent accordingly. that way it is much more clean and readable.

**Comment by christoball** [comment]
@jwhaney I made a commit to fix those issues

**Pull request merged**